meta-xt-prod-cockpit-rcar-driver-domain/recipes-qt/examples/dual-render-3d-part.bb

```diff
@@ -0,0 +1,39 @@
+DESCRIPTION = "Qt Quick 3D demo cluster"
+
+LICENSE = "CLOSED"
+
+inherit systemd
+
+RDEPENDS_${PN} = " \
+         qtquick3d \
+         qtbase \
+         qtdeclarative \
+"  
+
+FILES_${PN} = " \
+    ${bindir}/dual-render-3d-part \
+    ${systemd_unitdir}/system/3d-part.service \
+    ${sysconfdir}/eglfs.conf \
+"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += " \
+	file://3d-part.service \
+	git://git@gitpct.epam.com:rec-inv/dual-3d-part.git;branch=main \
```

Comment:
Please rename this repo `dual-3d-part` to `dual-render-3d-part.git`.
I have renamed repo on gitpct to `dual-render-3d-part`.

During an attempt to build domd I got an error:
```
ERROR: dual-render-3d-part-1.0-r0 do_fetch: Bitbake Fetcher Error: FetchError("Please set a valid SRCREV for url ['SRCREV_default_pn-dual-render-3d-part', 'SRCREV_default', 'SRCREV_pn-dual-render-3d-part', 'SRCREV'] (possible key names are git://git@gitpct.epam.com:rec-inv/dual-3d-part.git;branch=main, or use a ;rev=X URL parameter)", 'git://git@gitpct.epam.com:rec-inv/dual-3d-part.git;branch=main')
```